script.js

```diff
@@ -18,10 +20,104 @@ $(document).ready(function() {
     error: function() {
       alert("there has been an error...")
     }
-  })
-});
+  });
+
+
 
 function handleResponse(response) {
-  console.log(response);
-  // add stuff here!
-}
\ No newline at end of file
+  imgCount = response.data.length;
+  for (var i = 0; i <= response.data.length-1; i++) {
+    $(".instaimages").append("<img class=pics src='"+response.data[i].images.standard_resolution.url+"'/>");
```

Comment:
We didn't do this either, but after looking at Rafi's code, it seems like a good idea to place each image and caption in individual div's so that we can style each post.